server/src/main/java/com/epam/aidial/core/server/service/PublicationService.java

```diff
@@ -372,6 +378,48 @@ private void prepareAndValidatePublicationRequest(ProxyContext context, Publicat
         validateRules(publication);
     }
 
+    private void addCustomApplicationRelatedFiles(ProxyContext context, Publication publication) {
+        List<String> existingUrls = publication.getResources().stream()
+                .map(Publication.Resource::getSourceUrl)
+                .toList();
+
+        final String targetFolder;
+            {
+                String tempTargetFolder = publication.getTargetFolder();
+                int separatorIndex = tempTargetFolder.indexOf(ResourceDescriptor.PATH_SEPARATOR);
```

Comment:
Do you want to return prefix or postfix?

indexOf returns the first occurrence of the separator: a/b/c -> b/c

lastIndexOf: a/b/c -> a/b

Reply:
I think it is correct